tools/payments/cmd/disaster/main.go

```diff
@@ -152,15 +151,16 @@ func disaster(ctx context.Context, key, bucket, file string, shares [][]byte, en
 		log.Fatalf("failed to read attested recovery file: %v\n", err)
 	}
 
-	responseFile := "recovered-secrets.txt"
-	err = os.WriteFile(responseFile, recoveredData, 0644)
+	secretsFile := "recovered-secrets.txt"
```

Comment:
Well, technically this need not be a variable. 🙂 

```suggestion
	const secretsFile = "recovered-secrets.txt"
```